### PR TITLE
Create ui-c8y-1018-503-128-gainsight-add-more-properties-to-the-dashboards-custom-event-6872-7357.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by DawidSAG

Ticket: [MTM-60342](https://cumulocity.atlassian.net/browse/MTM-60342)
Original PR: [7357](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7357)

[MTM-60342]: https://cumulocity.atlassian.net/browse/MTM-60342?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1018-503-128-gainsight-add-more-properties-to-the-dashboards-custom-event-6872-7357.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: GAINSIGHT: Add more properties to the dashboards custom event (#6872) (#7357)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-128-gainsight-add-more-properties-to-the-dashboards-custom-event-6872-7357.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Gainsight: additional properties added to dashboards custom event
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-128-gainsight-add-more-properties-to-the-dashboards-custom-event-6872-7357.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60342
+version: 1018.503.128
+---
+GAINSIGHT: Add more properties to the dashboards custom event (#6872) (#7357)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-128-gainsight-add-more-properties-to-the-dashboards-custom-event-6872-7357.md:17]: The Claude AI suggested to use the following body:
```suggestion
To improve the data collected for product analytics in Gainsight, the custom event for dashboards has been extended. The dashboards custom event now includes additional properties such as the dashboard ID, the dashboard name, and the user ID of the user viewing the dashboard. This change enables more detailed analysis of dashboard usage patterns and user behavior. The impact of this change is limited to the data collected in Gainsight and does not affect the functionality or performance of dashboards in the application.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by rahulc8y — CHANGES_REQUESTED** [review]
I don't think we should include information about Gainsight-related changes in our customer facing changelog. After reviewing the last 1 year's changelog, I did not find similar information from other c8y products.

Please refer to the discussion on https://github.com/Cumulocity-IoT/c8y-docs/pull/2536

**Pull request closed without merging**